<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBook</class>
 <widget class="QDialog" name="addBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 239, 242);
border-radius: 8px;
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">color:black;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'georgia'; font-size:18pt; font-weight:700; color:#aa007f;&quot;&gt;ISBN :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ISBN">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 2px  solid;
     border-radius: 4px;
	font: 24px;color:black;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'georgia'; font-size:18pt; font-weight:700; color:#aa007f;&quot;&gt;Name :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Name">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Georgina</family>
          <pointsize>-1</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 2px  solid;
     border-radius: 4px;
	font: 24px;color:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'georgia'; font-size:18pt; font-weight:700; color:#aa007f;&quot;&gt;Author :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="chooseAuthBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton
{ 
	border-radius:15px;
	font: 16pt &quot;georgia&quot;;
	background-color:rgb(168, 0, 252);
	padding : 0 4px;
	border-radius:20px;
	color:rgb(255, 255, 0)
}


QPushButton:hover:!pressed

{
background:#52b788;
}
</string>
          </property>
          <property name="text">
           <string>Choose Author</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="author">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 2px  solid;
     border-radius: 4px;
	font: 24px;color:black;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'georgia'; font-size:18pt; font-weight:700; color:#aa007f;&quot;&gt;Genre :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="chooseGenreBtn">
          <property name="font">
           <font>
            <family>georgia</family>
            <pointsize>16</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{ 
	border-radius:15px;
	font: 16pt &quot;georgia&quot;;
	background-color:rgb(168, 0, 252);
	padding : 0 4px;
	border-radius:20px;
	color:rgb(255, 255, 0)
}


QPushButton:hover:!pressed

{
background:#52b788;
}
</string>
          </property>
          <property name="text">
           <string>Choose Genre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Genre">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox{
	background-color: rgb(255, 255, 255);
	 border: 2px  solid;
     border-radius: 4px;
	font: 24px;color:black;
}</string>
          </property>
          <item>
           <property name="text">
            <string>science</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>romantic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>philosophie</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>politic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sociale</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'georgia'; font-size:18pt; font-weight:700; color:#aa007f;&quot;&gt;Quantity :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="quantity">
        <property name="styleSheet">
         <string notr="true">QSpinBox{
	background-color: rgb(255, 255, 255);
	 border: 2px  solid;
     border-radius: 4px;
	font: 24px;color:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'georgia'; font-size:16pt; font-weight:700; color:#aa007f;&quot;&gt;Publisher :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="publisher">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 2px  solid;
     border-radius: 4px;
	font: 24px;color:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'georgia'; font-size:18pt; font-weight:700; color:#aa007f;&quot;&gt;Price :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="price">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	 border: 2px  solid;
     border-radius: 4px;
	font: 24px;color:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;georgia&quot;;
color: #e0777d;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#aa007f;&quot;&gt;Date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="date">
        <property name="styleSheet">
         <string notr="true">QDateEdit{
	background-color: rgb(255, 255, 255);
	 border: 2px  solid;
color:black;
     border-radius: 4px;
	font: 24px
}</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addBookBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton
{ 
	border-radius:15px;
	font: 16pt &quot;georgia&quot;;
	background-color:rgb(168, 0, 252);
	padding : 0 4px;
	border-radius:20px;
	color:rgb(255, 255, 0)
}


QPushButton:hover:!pressed

{
background:#52b788;
}
</string>
          </property>
          <property name="text">
           <string>Add Book</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>48</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>10</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'georgia'; font-size:14pt; font-weight:700; color:#aa007f;&quot;&gt;Book Description :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="description">
        <property name="styleSheet">
         <string notr="true">QTextEdit{
	background-color: rgb(255, 255, 255);
	 border: 3px  solid;
     border-radius: 4px;
	font: 24px
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="coverLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(143, 0, 214);
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;georgia&quot;;
color: #e0777d;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#aa007f;&quot;&gt;Book Cover&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>98</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="selectImgBtn">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>168</red>
                <green>0</green>
                <blue>252</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="PlaceholderText">
              <brush brushstyle="SolidPattern">
               <color alpha="128">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{ 
	border-radius:15px;
	font: 16pt &quot;georgia&quot;;
	background-color:rgb(168, 0, 252);
	padding : 0 4px;
	border-radius:20px;
	color:rgb(255, 255, 0)
}


QPushButton:hover:!pressed

{
background:#52b788;
}
</string>
          </property>
          <property name="text">
           <string>Select Image</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>88</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>48</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton
{ 
	border-radius:15px;
	font: 16pt &quot;georgia&quot;;
	background-color:rgb(168, 0, 252);
	padding : 0 4px;
	border-radius:20px;
	color:rgb(255, 255, 0)
}


QPushButton:hover:!pressed

{
background:#52b788;
}
</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
